### ISSUE TITLE
Persistent React list key error when using a list under WindupChildren

### ISSUE THREAD

**Issue by calummackervoy** [sgwilym/windups#9]
Apologies if I've missed something in the docs

I've had a persistent error message for the last few hours:

```
Warning: Each child in a list should have a unique "key" prop.
Check the render method of `WindupChildren`. See https://fb.me/react-warning-keys for more information.
```

my component is rendered like so:
```
<Container>
    <VisuallyHidden><ul className={styles.messageFeed}>{messagesUnread}</ul></VisuallyHidden>
    <div aria-hidden>
        <WindupChildren>
            <ul className={styles.messageFeed}>{messagesUnread}</ul>
        </WindupChildren>
    </div>
</Container>
```

each message in the list is rendered with a unique key as requested in the React docs

```
<Message message={message} key={message.id}/>
```

In debugging I found that `WindupChildren` is the cause of the issue (it's fine that the list is included under `VisuallyHidden` as well). I tested without the WindupChildren to confirm I've not done something wrong with the key

Love the library by the way, the project is a sort of text-based adenture game and windups really helps the UI come alive

**Comment by sgwilym** [comment]
Hi @calummackervoy, you haven't missed anything — this is a bug! Luckily it's an easy fix so I should have a patch out shortly.

And I'm thrilled that you're enjoying it. Text-based adventures drove me to build this thing, so I'm happy it's coming back around. 😄

**Comment by sgwilym** [comment]
Hopefully fixed in v1.1.6, let me know if you have any more trouble. Thanks!

**Comment by calummackervoy** [comment]
I confirm this fixes it, thanks !
